index.html

```diff
@@ -80,16 +80,28 @@ <h2>Add New Recipe</h2>
 
     <div id="image-container"></div>
 
-    <form id="search" type="submit">
+    <form id="search" type="add recipe">
       <input type="text" id="searchInput" placeholder="Search by title" />
-      <button type="submit" id="search-recipe">Search</button>
-      <button type="submit" onclick="sortRecipes()">Sort by Ingredients</button>
+      <button type="add recipe" id="search-recipe">Search</button>
```

Comment:
```suggestion
      <button type="submit" id="search-recipe">Search</button>
```
See previous comment about button's type.